src/cluster/dcommands/general/tag.ts

```diff
@@ -163,6 +167,9 @@ export class TagCommand extends GuildCommand {
                 }
             ]
         });
+
+        this.#docs = new BBTagDocumentationManager(cluster, 'tag');
```

Comment:
why are you not providing the `t` alias here?

Reply:
this `tag` is used to build the custom_id for the buttons/selects and is used to identify what should handle the interaction. Cant have multiple of those, and I wanted them to be ~3 chars. Will be using a similar system for handling interactions going forwards too I believe